.github/workflows/aws_tfhe_signed_integer_gpu_tests.yml

```diff
@@ -61,7 +61,8 @@ jobs:
     name: Setup instance (cuda-signed-integer-tests)
     runs-on: ubuntu-latest
     needs: should-run
-    if: github.event_name != 'pull_request' ||
+    if: (github.event_name == 'schedule' && github.repository == 'zama-ai/tfhe-rs') ||
+      github.event_name == 'workflow_dispatch' ||
```

Comment:
Same here, I don't think we want to change this.

Reply:
no we do, this runs the GPU tests on internal, I started the branch with the 4090 bench hence the name branch that is a bit confusing